app/views/users/_edit.html.erb

```diff
@@ -14,37 +14,34 @@
 <div id="general">
 	<div class="well">
 		<h2>General Information</h2>
+		<%= f.label :name, "Your username" %>
+		<%= f.text_field :name %>
+
+        <%= f.label :avatar, "Your current avatar" %>
 		<%= image_tag @user.avatar.url(:thumb) %>
-		<%if @user.avatar_file_name != nil%>
-		<%= f.label(:delete_avatar, 'Delete current avatar') %><%= f.check_box(:delete_avatar) %>
-		<%end%>
-	<div class="clearfix">
+		<% if @user.avatar_file_name != nil %>
+            <div class="checkbox">
+            <%= f.check_box(:delete_avatar) %> Delete current avatar?
+            </div>
+		<% end %>
+
 		<%= f.label :avatar, "Upload new avatar"%>
-		<%=f.file_field :avatar%>
-	</div>
-	
-	<div class="clearfix">
-		<%= f.label :name %>
-		<%= f.text_field :name %>
-	</div>
+		<%= f.file_field :avatar %>
+
 	
-    <h5><%=link_to "Change eMail/Password",{ :controller => "users", :action => "changepassword",:id => @user.id }%></h5>
+    <h5><%= link_to "Change eMail/Password", {controller: "users", action: "changepassword", id: @user.id} %></h5>
```

Comment:
Spaces look good after `{` and before `}`. Hound should pick that up, but apparently it doesn't like to read HTML.